test/deploy.js

```diff
@@ -52,7 +52,8 @@ test.beforeEach((t) => {
 test.serial('deployAll', (t) => {
```

Comment:
I think there is one more scenario to test (but maybe i cant see it here??), and I only discovered it when i was knee deep in the platform code. You can PUT  multiple interactions in one request via `PUT /answerspaces/name/interactions/1,2,3` where 1, 2,3 are the id's of the interactions to update, which must match to the objects in the `interactions` part of the payload.